tests/test_authenticators.py

```diff
@@ -46,6 +46,10 @@
 [auth:header_values]
 header = GSSAPI
 value = admin user
+
+[auth:header_comma_values]
+header = GSSAPI
+value = admin, user, space user
```

Comment:
what about someone mistyping with something like: "admin,user". Which is the expected behavior?